src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineSubGroupForCollector.java

```diff
@@ -0,0 +1,64 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.cornell.kfs.concur.ConcurConstants;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurStandardAccountingExtractDetailLine;
+
+/**
+ * Utility class for grouping together related SAE transactions with the same accounting data,
+ * and then partitioning them by payment code.
+ * 
+ * Only CASH (out of pocket) and CBCP (corporate card) SAE lines will be recorded by this class.
+ * The caller is expected to avoid sending in lines containing other payment codes.
+ */
+public class ConcurDetailLineSubGroupForCollector {
+
+    protected List<ConcurStandardAccountingExtractDetailLine> cashLines;
+    protected List<ConcurStandardAccountingExtractDetailLine> corporateCardLines;
+
+    public ConcurDetailLineSubGroupForCollector() {
+        this.cashLines = new ArrayList<>();
+        this.corporateCardLines = new ArrayList<>();
+    }
+
+    /**
+     * Convenience constructor that takes a Map-related key as input and then discards it;
+     * is only meant for use as a method reference to simplify certain Map operations.
+     */
+    public <T> ConcurDetailLineSubGroupForCollector(T mapKey) {

```

Comment:
A reference for this method was meant to be used as the last argument for the Map.computeIfAbsent() method, so that I wouldn't need a lambda expression to construct the argument. I'll see if I can clarify the intent of this or just replace its use case with a lambda expression instead.